home/import_helpers.py

```diff
@@ -184,14 +204,8 @@ def parse_file(
     return enumerate(fix_rows(read_rows(file_content)), start=2)
 
 
-def read_csv(file_content: bytes) -> Generator[dict[str, Any], None, None]:
-    try:
-        for row in csv.DictReader(StringIO(file_content.decode())):
-            yield {k.strip().lower(): v for k, v in row.items()}
-    except AttributeError:
-        raise ImportException(
-            "Invalid format. Please check that all row values have headers."
-        )
+def read_csv(file_content: bytes) -> Iterator[dict[str, str | None]]:
+    return csv.DictReader(StringIO(file_content.decode()))
 
 
 def read_xlsx(file_content: bytes) -> Generator[dict[str, Any], None, None]:
```

Comment:
Can we make this `Iterator[dict[str, str | None]]` (or maybe `Any` for the dict's value type) to match `read_csv()`?